app/models/foreman_tasks/triggering.rb

```diff
@@ -39,11 +40,11 @@ class Triggering < ActiveRecord::Base
     validate :correct_cronline, :if => Proc.new { |t| t.recurring? && t.input_type == :cronline }
 
     def self.new_from_params(params = {})
-      self.new(params).tap do |targeting|
+      self.new(params.except(:mode)).tap do |targeting|
         targeting.mode = params.fetch(:mode, :immediate).to_sym
         targeting.input_type = params.fetch(:input_type, :daily).to_sym
         targeting.end_time_limited = params[:end_time_limited] == "true"
-        targeting.start_at_raw = Time.now.strftime(TIME_FORMAT)
+        targeting.start_at_raw ||= Time.now.strftime(TIME_FORMAT)
```

Comment:
Could you please `s/targeting/triggering` as part of this pr? Don't know why I named it targeting when it's triggering